Project.toml

```diff
@@ -5,12 +5,13 @@ version = "0.4.0"
 
 [deps]
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 TOML = "1.0"
-julia = "1"
+julia = "1.6"
```

Comment:
why?

Reply:
Oops, this should read `1.5, 1.6`.
I tested this w/ `1.3` and it doesn't work.